Alpaca-Finance-Agents/Large-Position-Alert/src/agent.ts

```diff
@@ -0,0 +1,124 @@
+import BigNumber from 'bignumber.js'
+import { 
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType 
+} from 'forta-agent'
+import {
+  provideEventCheckerHandler,
+  FindingGenerator,
+  decodeParameters
+} from "forta-agent-tools";
+import { utils } from 'ethers';
+
+export const VAULT_ADDRESSES: string[] = [
+  "0x7C9e73d4C71dae564d41F78d56439bB4ba87592f", // BUSD
+  "0xbfF4a34A4644a113E8200D7F1D79b3555f723AfE", // ETH
+  "0x08FC9Ba2cAc74742177e0afC3dC8Aed6961c24e7", // BTCB
+  "0x08FC9Ba2cAc74742177e0afC3dC8Aed6961c24e7", // USDT
+  "0xf1bE8ecC990cBcb90e166b71E368299f0116d421", // ALPACA
+  "0x3282d2a151ca00BfE7ed17Aa16E42880248CD3Cd"  // TUSD
+];
+
+const VAULT_MAP: Map<string, bigint> = new Map([
+  [VAULT_ADDRESSES[0], BigInt(100000000000000000000000)], // 100,000 BUSD
+  [VAULT_ADDRESSES[1], BigInt(30000000000000000000)],     // 30 ETH
+  [VAULT_ADDRESSES[2], BigInt(3000000000000000000)],      // 3 BTCB
+  [VAULT_ADDRESSES[3], BigInt(100000000000000000000000)], // 100,000 USDT
+  [VAULT_ADDRESSES[4], BigInt(200000000000000000000000)], // 200,000 ALPACA
+  [VAULT_ADDRESSES[5], BigInt(100000000000000000000000)]  // 100,000 TUSD
+])
+
+export const workEventAbi: string = 'event Work(uint256 indexed id, uint256 loan)';
+const VAULT_IFACE: utils.Interface = new utils.Interface([workEventAbi]);
+export const workEventSig: string = VAULT_IFACE.getEvent('Work').format('sighash');
```

Comment:
I think you could replace this 3 lines.
```suggestion
export const workEventSig: string = "Work(uint256,uint256)"
```